bloom/generators/rosdebian.py

```diff
@@ -87,6 +87,7 @@ def fallback_resolver(key, peer_packages, rosdistro=self.rosdistro):
             releaser_history=releaser_history,
             fallback_resolver=fallback_resolver
         )
+        subs['ROSDistro'] = self.rosdistro
```

Comment:
The substitution case conventions are based on the Debian control file format and I think fully following that format would result in the field name `Ros-Distro` or `Rosdistro`.

I think either one of those, or completely abandoning the control format for `rosdistro` or `ros_distro` would be my preference.

Reply:
Fair point, switched to `Rosdistro` in 52bebba.